app/sagas/storeLabyrinthState.ts

```diff
@@ -0,0 +1,171 @@
+import { takeEvery, select, call } from 'redux-saga/effects';
```

Comment:
Do you think it's possible to easily gate this state writing logic on a config option (e.g. how traffic saving is handled)?  Seems like having the option to not write this out could be nice for folks not using the walker script.

Reply:
Yeah, this can be a config similar to saving the traffic.